tensorflow_datasets/scripts/templates/dataset.mako.md

```diff
@@ -165,6 +165,31 @@ ${builder.info.citation}
 % endif
 </%def>
 
+<%def name="display_figure(builder)">\
+<%
+  import requests
+  import os
+
+  config_name = os.path.split(builder.info.full_name)[-2]
+  ds_name_config = config_name.replace("\\", "/").replace("/", "_")
```

Comment:
Why is replace("\\", "/") required ? full_names returns the posix path.

Reply:
I think of windows also on adding this but i think its not require make changes